authentication/views/auth.py

```diff
@@ -33,8 +34,22 @@
 
 
 def index(request):
+    user_type_entry_message = {
```

Comment:
This is a todo - based on the user logged in we should greet them with a custom "header" and "content"